WordPress/src/test/java/org/wordpress/android/ui/reader/ReaderFileDownloadManagerTest.kt

```diff
@@ -59,7 +60,7 @@ class ReaderFileDownloadManagerTest {
         verify(request).setTitle(fileName)
     }
 
-    @Test
+    @Ignore @Test
```

Comment:
@ravishanker What is the reason behind ignoring this test? We don't typically want to ignore any tests, if they are incorrect, we should either fix them or remove them. If it really needs to be ignored, maybe a temporary edge case of sorts, we should add a comment explaining why it's ignored to provide context for other developers. We should also probably at least have a note, if not an explanation, in the PR description so the change doesn't go unnoticed/unreviewed in a large change set.

Reply:
That was another lint failure issue on an uninvoked test (test is never being called) .  First I tried to remove it, then I played safe.  
I removed it now.  Thanks